### ISSUE TITLE
Suggestion: RF/Cart Requirements for Maglev & RF accelerator.

### ISSUE THREAD

**Issue by shwible** [ryyanmapes/minecart-mod#23]
In this hypothetical feature:

Maglev tracks would only work/be fast for Maglev Carts.
Maglev tracks require X amount of RF/t in order for Maglev Carts to move. 

RF Accelerators would take both RF and a redstone signal.
As soon as the RF accelerator gets a redstone signal, any cart on top of the accelerator will speed up by an amount proportional to how much RF was stored in it. It will also use up all the RF stored. 

Honestly, any interaction with RF would be really cool.

**Comment by ryyanmapes** [comment]
Good idea. I'm waiting for more mods that use RF to be fleshed out a bit more before so I can correctly balance it though.

**Comment by shwible** [comment]
wdym? There's a bunch of mods for 1.16.5 that 

Mekanism
Immersive Engineering
Create Crafts and Addition (an addon that turns stress to rf)
Thermal Series(is still being developed but it's there)
PneumaticCraft(not really but they have a converter)
Advanced Rocketry
Industrial Foregoing
Electrodynamics (technically)

Extreme Reactors
RF Tools
Integrated Series
AE2
Refined Storage
